tests/playwright/sanity/modules/playing-cards/widget.test.js

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+import WpAdminPage from '../../../pages/wp-admin-page';
+import EditorSelectors from '../../../selectors/editor-selectors';
+
+test.describe( 'Playing Cards @playing-cards', () => {
+	test( 'Playing Cards widget', async ( { page }, testInfo ) => {
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		const editor = await wpAdmin.openNewPage();
+
+		await test.step( 'Add widget', async () => {
+			await editor.addWidget( 'playing-cards' );
+
+			const widget = await editor.getPreviewFrame().locator( EditorSelectors.playingCards.container );
```

Comment:
@pkniazevych You can use .first() here instead of the expect if having one widget is not the main thing you're testing.